sos/plugins/cri-o.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2018 Red Hat, Inc. Daniel Walsh <dwalsh@redhat.com>
```

Comment:
We'll need a different filename - `cri_o` most likely - Python modules are not allowed hyphens in the name.

Reply:
Or `crio`?

Reply:
Changed to crio.py